cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -0,0 +1,79 @@
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
```

Comment:
Yes. This is actually broken elsewhere already (in particular, hunt for
'enable_cluster_dns' and it's already broken). The problem, and I love
saying sentences like this, is that 'false' is true. So I think my code is
correct and/or we need to fix the escaping.

On Tue, Jan 20, 2015 at 11:02 AM, Joe Beda <notifications@github.com> wrote:

> In cluster/saltbase/salt/kube-addons/init.sls
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601#discussion-diff-23245950>
> :
>
> > @@ -0,0 +1,79 @@
> > +{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
>
> Exactly -- the explicit quoting makes it a string in yaml. That may be
> okay, but we should be consistent and eyes open here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601/files#r23245950>
> .
>